client/app/src/js/services/services.js

```diff
@@ -66,6 +66,29 @@ function Screenshots($http, $location, $window) {
 };
 Screenshots.$inject = ['$http','$location','$window'];
 
+function UserInfo($http){
```

Comment:
Spoke with Jackson.  Will rename to 'User', and will be remodeled to hold a user object that will come back from authenticating.  

The main purpose is to have the user information stored in a factory so all controllers can have access to the current user by injecting the factory in.